docs/bridge/sequences.md

```diff
@@ -0,0 +1,62 @@
+## Deposit
+
+Bridge tokens from rootchain to childchain via deposit.
+
+```mermaid
+sequenceDiagram
+	User->>Edge: deposit
+	Edge->>RootERC20.sol: approve(RootERC20Predicate)
+	Edge->>RootERC20Predicate.sol: deposit()
+	RootERC20Predicate.sol->>RootERC20Predicate.sol: mapToken()
+	RootERC20Predicate.sol->>StateSender.sol: syncState(MAP_TOKEN_SIG)
```

Comment:
When we do a first deposit, we are mapping the tokens for the first time, so there are two state sync events, one with the `MAP_TOKEN_SIG` to map the tokens on the child predicate, and the second one that actually carries the deposit (`DEPOSIT_SIG`), so I suggest a small change to represent this on the diagram:

```suggestion
	RootERC20Predicate.sol->>StateSender.sol: syncState(MAP_TOKEN_SIG)
	RootERC20Predicate.sol-->>Edge: TokenMapped Event
	StateSender.sol-->>Edge: StateSynced Event to map tokens on child predicate
	RootERC20Predicate.sol->>StateSender.sol: syncState(DEPOSIT_SIG)
	StateSender.sol-->>Edge: StateSynced Event to deposit on child chain
	Edge->>User: ok
	Edge->>StateReceiver.sol:commit()
	StateReceiver.sol-->>Edge: NewCommitment Event
	Edge->>StateReceiver.sol:execute()
	StateReceiver.sol->>ChildERC20Predicate.sol:onStateReceive()
	ChildERC20Predicate.sol->>ChildERC20.sol: mint()
	StateReceiver.sol-->>Edge:StateSyncResult Event
```